### DISCUSSION TITLE
After reboot panel dose not open

### DISCUSSION THREAD

**Discussion by Ragemalik** [vilhelmprytz/pterodactyl-installer#213] [Q&A]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behaviour:
1. Run the command sudo reboot
2. After reboot try to go to your panel

**Expected behaviour**
You should be brought up to the panel

**OS (please complete the following information):**
 - ubmutu 20.04
 - installed wings and pannel from here

**Comment by Linux123123** [comment]
The script installs a service, which always runs. Your panel wouldn't work at beginning if that service wasn't started. Without any additional logs it is hard to say what is exactly wrong. Come to our [discord server](https://pterodactyl-installer.se/discord) for support :)

**Comment by vilhelmprytz** [comment]
I moved this to discussion since it is not a problem with the script. As @Linux123123 mentioned, refer to the Discord server for support.
